### PR TITLE
Update Web SDK licensing info

### PR DESCRIPTION
**Pull request by kopijunkie**
# Problem
The current Web SDK MIT licensing presents some confusion with regards to our contractual terms of use. We want to simply move to our own proprietary licence.

# Solution
- Remove mentions of Onfido SDK being under MIT licence from README
- Replaced copy in LICENSE to new copy

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [x] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1296-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1296-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1296-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1296-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1296-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1296-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1296-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1296-pr-onfido-sdk-ui-onfido.surge.sh

File: LICENSE [hunk 1]
```diff
@@ -1,21 +1 @@
...
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-THE SOFTWARE.
+Please see the terms of your Onfido Services Agreement which you are bound by with respect to your use of the Onfido SDK. To the extent the Onfido SDK includes copies or substantial portions of software licenced under third party terms and conditions this will be set out at https://github.com/onfido/onfido-sdk-ui/blob/master/dist/onfido.LICENSES.txt and https://github.com/onfido/onfido-sdk-ui/blob/master/dist/demo.LICENSES.txt. In the event of any inconsistency between the Onfido Services Agreement and any third party terms and conditions, the third party terms and conditions shall take precedence over the rights and restrictions granted in the Onfido Services Agreement solely in respect of such third party dependencies set out at https://github.com/onfido/onfido-sdk-ui/blob/master/dist/onfido.LICENSES.txt and https://github.com/onfido/onfido-sdk-ui/blob/master/dist/demo.LICENSES.txt.
```
Comment (stefaniacardenas) [inline LICENSE:1]: At the link below you will only find the license comments that are included in third-party dependencies codebases.
https://github.com/onfido/onfido-sdk-ui/blob/master/dist/onfido.LICENSES.txt
https://github.com/onfido/onfido-sdk-ui/blob/master/dist/demo.LICENSES.txt

The full list of third party licenses can be found here https://github.com/onfido/onfido-sdk-ui/blob/master/licenses.json.

Maybe we should create a folder and place all the files related to licenses there.

**Review by stefaniacardenas — CHANGES_REQUESTED** [review]
The `package.json` still has "MIT" as a value for `"license"`. Is that intended? I left a comment about an additional file that should be added to LICENSE file

**Comment by kopijunkie** [comment]
> The `package.json` still has "MIT" as a value for `"license"`. Is that intended? I left a comment about an additional file that should be added to LICENSE file

Good spot. Will update that.

File: LICENSE (hunk 1 above)
Comment (kopijunkie) [reply LICENSE:1]: Yeah, looks like it'll be better to consolidate everything in a folder. Currently the comments get generated in `dist` while the full list is in the root folder. I'll raise a ticket to address that.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1296-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1296-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1296-pr-onfido-sdk-ui-onfido.surge.sh

File: package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "type": "git",
     "url": "https://github.com/onfido/onfido-sdk-ui.git"
   },
-  "license": "MIT",
+  "license": "SEE LICENSE IN onfido-sdk-ui/LICENSE",
```
Comment (stefaniacardenas) [inline package.json:10]: Minor: I think this should be
```suggestion
  "license": "SEE LICENSE IN ./LICENSE",
```

**Review by stefaniacardenas — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (kopijunkie) [reply LICENSE]: As discussed created a separate ticket for consolidating the files.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1296-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1296-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1296-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1296-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1296-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1296-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**